src/content/ui/adaptive-responsive/best-practices.md

```diff
@@ -0,0 +1,240 @@
+than a traditional desktop UI due, in part,
+to the lack of input accelerators like right-click,
+scroll wheel, or keyboard shortcuts.
+
+One way to approach this challenge is to focus initially
+on a great touch-oriented UI. You can still do most of
+your testing using the desktop target for its iteration speed.
+But, remember to switch frequently to a mobile device to
+verify that everything feels right.
+
+After you have the touch interface polished, you can tweak
+the visual density for mouse users, and then layer on all
+the additional inputs. Approach these other inputs as
+accelerator—alternatives that make a task faster.
+The important thing to consider is what a user expects
+when using a particular input device,
+and work to reflect that in your app.
+
+## Implementation details
+
+### Don't lock the orientation of your app.
+
+An adaptive app should look good on windows of
+different sizes and shapes. While locking an app
+to portrait mode on phones can help narrow the scope
+of a minimum viable product, it can increase the
+effort required to make the app adaptive in the future.
+
+For example, the assumption that phones will only
+render your app in a full screen portrait mode is
+not a guarantee. Multi window app support is becoming common,
+and foldables have many use cases that work best with
+multiple apps running side by side.
+
+If you absolutely must lock your app in portrait mode (but don't),
+use the `Display` API instead of something like `MediaQuery`
+to get the physical dimensions of the screen.
+
+To summarize:
+
+  * Locked screens can be [an accessibility issue][] for some users
+  * Android large format tiers require portrait and landscape
+    support at the [lowest level][].
+  * Android devices can [override a locked screen][]
+  * Apple guidelines say [aim to support both orientations][] 
+
+[an accessibility issue]: https://www.w3.org/WAI/WCAG21/Understanding/orientation.html
+[aim to support both orientations]: https://www.w3.org/WAI/WCAG21/Understanding/orientation.html
+[lowest level]:  {{site.android-dev}}/docs/quality-guidelines/large-screen-app-quality#T3-8
+[override a locked screen]: {{site.android-dev}}/guide/topics/large-screens/large-screen-compatibility-mode#per-app_overrides
+
+### Avoid orientation-based layouts
+
+Avoid using `MediaQuery`'s orientation field
+or `OrientationBuilder` to switch between
+different app layouts. This is similar to the
+guidance of not checking device types to determine
+screen size. The device's orientation also doesn't
+necessarily inform you of how much space your app window has.
+
+Instead, use `MediaQuery`'s `sizeOf` or `LayoutBuilder`,
+as discussed in the [General approach][] page.
+Then use adaptive breakpoints like the ones that
+[Material][] recommends.
+
+[General approach]: /ui/adaptive-responsive/general#
+[Material]: https://m3.material.io/foundations/layout/applying-layout/window-size-classes
+
+### Don't gobble up all of the horizontal space
```

Comment:
The word "gobble" is fun, but for the purposes of this article, it might not be the most approachable. Perhaps "take all" is sufficient?